src/assets/fabricator/styles/components/layout.css

```diff
@@ -0,0 +1,67 @@
+.f-Layout {
+  height: 100%;
+}
+
+.f-Layout.f-is-active {
```

Comment:
(Plus it seems like it would make it easier, later, to discern very easily between fabricator CSS and project CSS in debug)